src/fklearn/preprocessing/schema.py

```diff
@@ -0,0 +1,121 @@
+import functools
+import inspect
+import pandas as pd
+import toolz
+
+from typing import Any, Callable, Dict, List, Optional, Union
+
+from ..types import LearnerLogType, LearnerReturnType
+
+
+@toolz.curry
+def feature_duplicator(df: pd.DataFrame,
+                       columns_to_duplicate: Optional[List[str]] = None,
+                       columns_mapping: Optional[Dict[str, str]] = None,
+                       preffix: Optional[str] = None,
+                       suffix: Optional[str] = None) -> LearnerReturnType:
+    """
+    Duplicates some columns in the dataframe
+
+    Parameters
+    ----------
+    df: pandas.DataFrame
+        A Pandas' DataFrame with columns_to_duplicate columns
+
+    columns_to_duplicate: list of str
+        List of columns names
+
+    columns_mapping: int (default None)
+        Mapping of source columns to destination columns
+
+    preffix: int (default None)
+        Preffix to add to columns to duplicate
+
+    suffix: int (default None)
+        Suffix to add to columns to duplicate
+
+    Returns
+    ----------
+    increased_dataset : pandas.DataFrame
+        A dataset with repeated columns
+    """
+
+    stackname = inspect.stack()[0].function
```

Comment:
It isn't very clear for me what this do just reading the code. What about leaving at least a comment or a more descriptive variable name? Also, the use of `inspect` and `eval` is pretty unsafe, isn't there another way of achieving the same thing here?